### ISSUE TITLE
Missing content

### ISSUE THREAD

**Issue by tussle** [rach0012/humanRL_gym_ple#1]
This repository seems to be missing a lot of files including the code defining the various games.  Can you make that available?

**Comment by tussle** [comment]
sorry; my mistake.

**Comment by tussle** [comment]
Sorry for all the confusion but I'm unable to import originalGame and can't find any such game in this repo or within the PLE code.


Hopefully you can help

Thanks

**Comment by rachit-dubey** [comment]
Hi, thanks for contacting me. There are two main repos for our paper. The first one is https://github.com/rach0012/humanRL_prior_games which contains the codes for the games we built on which we trained the RL agent (the games are the modification of the popular PLE repo). The current repo is the modification of gym-ple, that helps to use our games as openAI environments to run RL agents on. 

So all the games are contained the humanRL_prior_games repo. Are you able to access this - https://github.com/rach0012/humanRL_prior_games/tree/master/ple/games/originalgame? This one has the original game. 

Thanks!
